src/Avalonia.DesignerSupport/DesignerAssist.cs

```diff
@@ -86,67 +86,11 @@ private static void SetScalingFactor(double factor)
         private static void UpdateXaml2(Dictionary<string, object> dic)
```

Comment:
Any chance we can rename this method? Or just combine it with `UpdateXaml`?

Reply:
It's a part of the public API or our current previewer - https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.DesignerSupport/DesignerApi.cs#L34

Can't touch that without breaking VS extension.

Reply:
That makes sense. Kinda sucks that we got stuck with this naming though..

Reply:
It's a common way of ABI versioning. Just look at visual studio and countless `IVSProject9001` interfaces.